ArduPlane/system.cpp

```diff
@@ -149,7 +149,8 @@ void Plane::init_ardupilot()
 
     // setup frsky
 #if FRSKY_TELEM_ENABLED == ENABLED
-    frsky_telemetry.init(serial_manager);
+    // setup frsky, and pass a number of parameters to the library
+    frsky_telemetry.init(serial_manager, (uint8_t *)&control_mode);
```

Comment:
don't pass it in at all, use an accessor in set_mode() change it in the frsky library.